client/resources/languages/fr.json

```
@@ -5,48 +5,48 @@
   "login error": "Nom d‘utilisateur ou mot de passe incorrect",
 
   "suggestions": "Suggestions",
-  "suggestions for": "Suggestions for",
-  "refresh infos": "Refresh infos",
+  "suggestions for": "Suggestions pour",
+  "refresh infos": "Actualiser les infos",
   "delete item": "Supprimer",
   "confirm delete": "Es-tu sûr(e) ?",
   "search": "Rechercher",
-  "search for": "Search for",
-  "search or add": "Rechercher ou ajouter",
+  "search for": "Chercher",
+  "search or add": "Chercher ou ajouter",
   "load more": "Afficher plus",
   "nothing found": "Aucun résultat trouvé",
 
-  "calendar": "Calendar",
+  "calendar": "Calendrier",
   "trending": "Tendances",
   "upcoming": "Prochainement",
-  "now-playing": "Now Playing",
+  "now-playing": "En cours de lecture",
   "tv": "Séries TV",
-  "episodes": "Episodes",
+  "episodes": "Épisodes",
   "movies": "Films",
-  "movie": "Movie",
+  "movie": "Film",
   "last seen": "Dernier film/épisode vu",
-  "own rating": "Own Rating",
-  "title": "Title",
-  "release": "Release",
-  "tmdb rating": "TMDb Rating",
-  "imdb rating": "IMDb Rating",
+  "own rating": "Ma note",
```

C'est utilisé dans le menu permettant le classement par type.
```suggestion
  "own rating": "Mon classement",
```